django/forms/fields.py

```diff
@@ -251,7 +254,8 @@ def get_bound_field(self, form, field_name):
         Return a BoundField instance that will be used when accessing the form
         field in a template.
         """
-        return BoundField(form, self, field_name)
+        BoundFieldClass = self.bound_field_class or form.bound_field_class
```

Comment:
nitpick: I'm personally not sure about having this as camel case. It feels like there's an actual class `BoundFieldClass`, whereas `bound_field_class` is a variable. Not sure, but it's minor.

Reply:
Good point. Thank you.